files/default/setup-ephemeral-drives.sh

```diff
@@ -26,7 +26,7 @@ function exec_command() {
   _exit_code=$?
 
   # Do not set RC=1 if error says that changes have been written but a reboot is required to inform the kernel
-  [[ $_exit_code -ne 0 && $(echo "${_command_output}" | grep -i "you should reboot now") ]] && RC=1
+  [[ $_exit_code -ne 0 && $(echo "${_command_output}" | grep -vi "you should reboot now") ]] && RC=1
```

Comment:
grep will return 1 if it does't find any match and the overall condition will result to false right?

Reply:
check improved